source/val/validate_builtins.cpp

```diff
@@ -2507,8 +2507,8 @@ spv_result_t BuiltInsValidator::ValidateSingleBuiltInAtDefinition(
     case SpvBuiltInMeshViewIndicesNV:
     case SpvBuiltInBaryCoordNV:
     case SpvBuiltInBaryCoordNoPerspNV:
-    case SpvBuiltInFragmentSizeNV:
-    case SpvBuiltInInvocationsPerPixelNV:
+    case SpvBuiltInFragmentSizeNV: // alias SpvBuiltInFragSizeEXT
```

Comment:
This is producing a formatting error. Please clang-format this file.

Reply:
Fixed. 
FWIW - the CI logs are completely useless. I couldn't get any log files or anything. I'd click the links and it'd say the files didn't exist. Maybe you need to be a Google employee to seem them or something, but it would be nice if they could be made useful for other people too!

Reply:
Yes, we apologize for that and are working on getting the logs public again. They were briefly available publicly some weeks ago.